### PR TITLE
bug/380_missing_template_mongo

### PR DESCRIPTION
**Pull request by frbattid**
* Fixes issue #380 
* No tests required
* Assignee @fgalan but LGTM from @gtorodelvalle is also required

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,16 +28,16 @@
 
 ##<a name="section1"></a>What is Cygnus
 
-Cygnus is a connector in charge of persisting [Orion](https://github.com/telefonicaid/fiware-orion) context data in certain configured third-party storages, creating a historical view of such data. In other words, Orion only stores the last value regarding an entity's attribute, and if an older value is required then you will have to persist it in other storage, value by value, using Cygnus.
```
Comment (frbattid) [inline README.md]: I don't know why this edition has been included... there is no change on the text...

File: README.md [hunk 2]
```diff
@@ -28,16 +28,16 @@
 
 ##<a name="section1"></a>What is Cygnus
 
-Cygnus is a connector in charge of persisting [Orion](https://github.com/telefonicaid/fiware-orion) context data in certain configured third-party storages, creating a historical view of such data. In other words, Orion only stores the last value regarding an entity's attribute, and if an older value is required then you will have to persist it in other storage, value by value, using Cygnus.
+Cygnus is a connector in charge of persisting [Orion](https://github.com/telefonicaid/fiware-orion) context data in certain configured third-party storages, creating a historical view of such data. In other words, Orion only stores the last value regarding an entity's attribute, and if an older value is required then you will have to persist it in other storage, value by value, using Cygnus.
 
-Cygnus uses the subscription/notification feature of Orion. A subscription is made in Orion on behalf of Cygnus, detailing which entities we want to be notified when an update occurs on any of those entities attributes.
```
Comment (frbattid) [inline README.md]: I don't know why this edition has been included... there is no change on the text...

File: README.md [hunk 3]
```diff
@@ -28,16 +28,16 @@
...
+Cygnus uses the subscription/notification feature of Orion. A subscription is made in Orion on behalf of Cygnus, detailing which entities we want to be notified when an update occurs on any of those entities attributes.
 
 Internally, Cygnus is based on [Apache Flume](http://flume.apache.org/). In fact, Cygnus is a Flume agent, which is basically composed of a source in charge of receiving the data, a channel where the source puts the data once it has been transformed into a Flume event, and a sink, which takes Flume events from the channel in order to persist the data within its body into a third-party storage.
 
-Current stable release is able to persist Orion context data in:
```
Comment (frbattid) [inline README.md]: I don't know why this edition has been included... there is no change on the text...

File: README.md [hunk 4]
```diff
@@ -28,16 +28,16 @@
...
+Current stable release is able to persist Orion context data in:
 
-* [HDFS](http://hadoop.apache.org/docs/current/hadoop-project-dist/hadoop-hdfs/HdfsUserGuide.html), the [Hadoop](http://hadoop.apache.org/) distributed file system.
-* [MySQL](https://www.mysql.com/), the well-know relational database manager.
```
Comment (frbattid) [inline README.md]: I don't know why this edition has been included... there is no change on the text...

**Comment by frbattid** [comment]
This was planned for 0.8.1, but again, it has no sense to release something (0.8.0 release) by knowing there is something wrong :)

File: README.md (hunk 1 above)
Comment (fgalan) [inline README.md]: Maybe some newline or someting like that has changed. Irrelevant.

NTC

File: README.md (hunk 2 above)
Comment (fgalan) [inline README.md]: NTC

File: README.md (hunk 3 above)
Comment (fgalan) [inline README.md]: NTC

File: README.md (hunk 4 above)
Comment (fgalan) [inline README.md]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**